# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: xpcom/io/moz.build
# Commit: ef46ec074d6a
# Full Hash: ef46ec074d6aba10d6e4342e305f36ead6996ddb
# Author: Honza Bambas <honzab.moz@firemni.cz>
# Date: 2018-06-07 21:50:48
# Regressor Bug: 1413868
# File Overlap Count: 1
# Description:
#   Bug 1413868. r=valentin
# ==============================================================================

diff -r 14f7dab3a9bd -r ef46ec074d6a xpcom/io/moz.build
--- a/xpcom/io/moz.build	Wed Jun 06 14:50:17 2018 -0700
+++ b/xpcom/io/moz.build	Thu Jun 07 13:56:16 2018 -0400
@@ -84,6 +84,7 @@
 
 EXPORTS.mozilla += [
     'Base64.h',
+    'FilePreferences.h',
     'InputStreamLengthHelper.h',
     'InputStreamLengthWrapper.h',
     'NonBlockingAsyncInputStream.h',
@@ -97,6 +98,7 @@
     'Base64.cpp',
     'crc32c.c',
     'FileDescriptorFile.cpp',
+    'FilePreferences.cpp',
     'InputStreamLengthHelper.cpp',
     'InputStreamLengthWrapper.cpp',
     'NonBlockingAsyncInputStream.cpp',